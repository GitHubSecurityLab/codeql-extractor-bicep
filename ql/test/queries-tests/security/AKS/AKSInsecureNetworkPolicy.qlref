security/CWE-668/AKSInsecureNetworkPolicy.ql
