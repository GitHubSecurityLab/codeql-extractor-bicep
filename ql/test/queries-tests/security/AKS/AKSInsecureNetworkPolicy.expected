| aks-security-examples.bicep:2:1:30:1 | ManagedContainerResource | AKS cluster is configured with an insecure or missing network policy, which may allow unwanted pod-to-pod communication. |
| aks-security-examples.bicep:32:1:62:1 | ManagedContainerResource | AKS cluster is configured with an insecure or missing network policy, which may allow unwanted pod-to-pod communication. |
| app.bicep:287:1:307:1 | ManagedContainerResource | AKS cluster is configured with an insecure or missing network policy, which may allow unwanted pod-to-pod communication. |
| app.bicep:310:1:330:1 | ManagedContainerResource | AKS cluster is configured with an insecure or missing network policy, which may allow unwanted pod-to-pod communication. |
